pkg/sql/descmetadata/metadata_updater.go

```diff
@@ -145,3 +145,16 @@ func (mu metadataUpdater) SwapDescriptorSubComment(
 	)
 	return err
 }
+
+// DeleteSchedule implement scexec.DescriptorMetadataUpdater.
+func (mu metadataUpdater) DeleteSchedule(ctx context.Context, scheduleID int64) error {
+	_, err := mu.ie.ExecEx(
+		ctx,
+		"delete-schedule",
+		mu.txn,
+		sessiondata.InternalExecutorOverride{User: security.RootUserName()},
+		"DELETE FROM system.scheduled_jobs WHERE schedule_id = $1",
```

Comment:
drop schedule perhaps?

Reply:
i avoided using DROP SCHEDULE as it would allow users to DROP schedules as well. currently we've explicitly banned that. (also this was mostly form following the `deleteSchedule` code in the `sql` package; we have no hooks for this ... yet ...